.github/workflows/push.yml

```diff
@@ -92,6 +94,7 @@ jobs:
           Initialize-AzOpsEnvironment
           Import-PSFConfig -Path settings.json -Schema MetaJson
           $diff = Get-Content -Path /tmp/diff.txt
+          $diffdeletedfiles = Get-Content -Path /tmp/diffdeletedfiles.txt
```

Comment:
The deleteset should already be in the changeset, why is this necessary?

Reply:
@daltondhcp  - As we discussed, this is regarding the delete content which is specific to the resource deletion module as the file is not present.